### PR TITLE
Configure setup via setup.cfg, and refresh README content

### PR DESCRIPTION
**Pull request by twidi**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -54,21 +37,50 @@ is an example URL conf::
...
+------------------
+
+============== ==================
+Django version Python versions
+============== ==================
+1.11           2.7, 3.4, 3.5, 3.6
+2.0            3.4, 3.5, 3.6
+============== ==================
+
+All library versions to use for old django/python support
```
Comment (jdufresne) [inline README.rst]: I would capitalize "Django" and "Python" here. They are proper nouns.

**Comment by twidi** [comment]
It's a work done by @ewjoachim and myself when preparing our talk do pyconfr last year (you can find the slides here https://twidi.github.io/python-packaging-talk/fr still needed to be translated). Maybe Joachim can give you more links than the only one I have: https://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files

**Comment by jdufresne** [comment]
I see there are some docs here: https://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files

Cool. Thanks for showing me this technique.

File: README.rst (hunk 1 above)
Comment (twidi) [reply README.rst]: Done

**Comment by ewjoachim** [comment]
I second what @twidi said :) I don't have a better link than the one you both shared or if you're that much into code, [the PR](https://github.com/pypa/setuptools/pull/862) that made it possible.

As a final note, it seems the wind keeps blowing, now towards pyproject.toml (see [PEP 518](https://www.python.org/dev/peps/pep-0518/)), so same advantages as setup.cfg in terms of readability by a machine etc, but in a better format with better specifications. At some point, someone will merge in setuptool the possibility to do the same in pyproject.toml as can be done in setup.cfg, and then we should be stable. Maybe.

But we'll definitely be better off than with everything in setup.py.

BTW, why was this PR closed without merging ?

**Comment by twidi** [comment]
@ewjoachim sorry very very (very very) late answer but: merged because PR branch deleted before pushing master (or develop) so Github couldn't see the head of this branch (which didn't exist anymore for them) on the base branch... so marked it at closed, but really it was merged

**Pull request closed without merging**